INFO     backend.layers.business.business:business.py:515 {'message': 'ingesting dataset', 'collection_version_id': 48341ad0-b702-40ba-a581-ec64279995ae, 'url': 'http://fake.url', 'current_dataset_version_id': None}
INFO     backend.layers.business.business:business.py:515 {'message': 'ingesting dataset', 'collection_version_id': 48341ad0-b702-40ba-a581-ec64279995ae, 'url': None, 'current_dataset_version_id': b9b29b0d-921f-4888-9051-ba25b96841c4}
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:logger.py:26 Start create_artifact
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:process_logic.py:91 Uploaded [a3e5114d-b99a-42fa-9fde-3d713f4211ea/raw.h5ad] to s3://artifact_bucket/a3e5114d-b99a-42fa-9fde-3d713f4211ea/raw.h5ad
INFO     processing:process_logic.py:93 Updated database with raw_h5ad.
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:logger.py:28 Complete create_artifact
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:dataset_metadata_update.py:244 Main: Starting thread for h5ad update
INFO     processing:dataset_metadata_update.py:264 Main: Starting thread for rds update
ERROR    processing:dataset_metadata_update.py:302 Cannot find cxg artifact uri for b9b29b0d-921f-4888-9051-ba25b96841c4.
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:process_logic.py:45 Updating processing status